packages/api/src/task/scheduler.ts

```diff
@@ -371,6 +376,28 @@ export class TaskScheduler {
     return task;
   }
 
+  private deleteCredentials(
```

Comment:
Can we name this function more clearly? I was confused about it thinking it would delete credentials of any task on the first update, which is not the case.

WDYT of something like `deleteTranscodeCredsOnCompletion` or sth like that?

Reply:
I named it `deleteCredentialsOnCompletion`, because I think it's not only related to `Transcode`, it just removes credentials from any `objectStoreUrl`.